draft-ietf-capport-architecture.xml

```diff
@@ -249,6 +248,14 @@
         portal web application, and (c) legacy devices must continue to be
         supported.
         </t>
+        <t>
+        If User Equipment supports Captive Portal API, it MUST validate API server TLS certificate
+        (see <xref target="RFC2818"/>). An Enforcement device SHOULD allow User Equipment to
+        connections to OCSP responders, CRLs and NTP servers while still in captive mode (see
+        Section 4.1 in <xref target="I-D.ietf-capport-api"/>) to validate the server certificate.
```

Comment:
Maybe there is a simpler approach here:

```suggestion
        (see <xref target="RFC2818"/>). An Enforcement device SHOULD allow access to
        any services that User Equipment could need to contact to perform certificate validation,
        such as OCSP responders, CRLs, and NTP servers; see Section 4.1 of 
        <xref target="I-D.ietf-capport-api"/> for more information.
```

This avoids being definitive, but provides the same examples..